CMSgov/onemac
# OY2 20217: Consolidate a list of all messages and correlating forms for Submission and Package View

Story: https://qmacbis.atlassian.net/browse/OY2-20217
Endpoint: See github-actions bot comment

### Details
A list of the messages and form messages, etc will be obsolete pretty much the moment it is created.... devise an automagic form describer instead - at least for the Package View, which can change more.

### Changes
- cleaned up SOME of the error messaging in the forms 
- created "forms-describe" - a hidden page which describes package view forms
- removed all the complicated ID existence checking carried over from Submission View - Package View will have a better concept of "parentage" so should not need the ability to perform extra checks on ID existence based on format alone.

### Implementation Notes
- it would be super great if this did not exist in the application, if it could be some sort of admin tool that references the config files and supporting messaging, instead...

### Test Plan
1. Login to the Application as a System Admin User
2. Navigate to forms-describe using the URL address
3. see all the goodies!
